app/controllers/application_controller.rb

```diff
@@ -5,5 +5,8 @@ class ApplicationController < ActionController::Base
 
   def current_user
     @current_user ||= User.find(session[:user_id]) if session[:user_id]
+    rescue ActiveRecord::RecordNotFound
```

Comment:
@josh-works can you explain what is happening here? Im not sure I understand. Something to do with resetting the session?